
Header header

string head
float64 timeUTC

float64 latitude
string northOrSouth
float64 longitude
string eastOrWest

int16 GPSstatus
int16 satelliteNum
float64 HDOP
float64 altitude
string altitudeUnit

float64 geiod
string geiodUnit
int16 rtkAge
string rtkID
